### PR TITLE
Create the Report Details Page

### PR DESCRIPTION
**Pull request by jasperhuangg**
### Details
This PR adds the Report Details Page for default rooms and implements the ability to set notification preferences for a chatroom via a Picker.

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/161785

### Tests
1. Navigate to a default room.
2. Clicked on the header for the default room. Verify it pulls up the report details page, and that it renders correctly.
3. Change the notification preference. Verified that the Picker is updated to reflect the new notification preference, and that the API call to `Report_UpdateNotificationPreference` succeeds.
4. Click on the "Members" menu item. Verify that the members list renders correctly.

### QA Steps
Same as the above tests

### Tested On

- [x] Web
- [ ] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web
https://user-images.githubusercontent.com/31285285/123371437-5a417180-d5b4-11eb-9136-d0e960fc5388.mp4

#### Desktop
https://user-images.githubusercontent.com/31285285/123371868-1b5feb80-d5b5-11eb-9a51-9ccbf48a860f.mp4

#### iOS
https://user-images.githubusercontent.com/31285285/123374490-c377b380-d5b9-11eb-9d57-104f52d376a9.mp4

#### Android
https://user-images.githubusercontent.com/31285285/123375089-c7580580-d5ba-11eb-951e-6192bb38a0b2.mp4

cc @yuwenmemon

### CODE REVIEW

File: src/languages/en.js [hunk 1]
```diff
@@ -146,6 +147,13 @@ export default {
         split: ({amount}) => `Split ${amount}`,
         choosePaymentMethod: 'Choose payment method:',
     },
+    reportDetailsPage: {
+        notificationPreferencesDescription: 'How often should we notify you when there are new messages to catch up on in this room?',
+        always: 'Always',
+        daily: 'Daily',
+        mute: 'Mute',
+        members: 'Members',
+    },
```
Comment (jasperhuangg) [inline src/languages/en.js:156]: Spanish translations for these were not included in the Design Doc. Reaching out in #espanol for help.

**Comment by jasperhuangg** [comment]
@ctkochan22 Added some missing translations and took it off hold, should now be good for a review, thanks!

**Review by ctkochan22 — APPROVED** [review]

**Comment by isagoico** [comment]
@jasperhuangg @ctkochan22 Heyo! We were not able to test this on our side. We're currently unable to add people to the workspace and create the chat rooms (On our expensifail domain).

**Comment by isagoico** [comment]
Tested this and it was a pass on Android, mWeb and Web, will get someone to test iOS and Desktop🎉(We were not sure on how to access rooms but Yuwen clarified here https://github.com/Expensify/Expensify.cash/pull/3766#issuecomment-872560520)  


image

**Comment by puneetlath** [comment]
Heads up @jasperhuangg @ctkochan22! Just FYI: this PR caused a bug here: https://github.com/Expensify/App/issues/11982

**Pull request merged**